### PR TITLE
feat(hostinfo/installed_services): Ported a waldo required ohai check…

### PR DESCRIPTION
**Pull request by kaustavha**
More or less direct port of
https://github.com/rackerlabs/ohai-plugins/blob/master/plugins/installed_services.rb

### CODE REVIEW

**Comment by rphillips** [comment]
the tabs are off on this file.

**Comment by kaustavha** [comment]
That was strange, reindented. Thanks!

**Comment by rphillips** [comment]
2 spaces please... looks like kernel_modules.lua can be removed from the PR

**Comment by kaustavha** [comment]
Strange, for some reason wstorm decided to forget what the indentation rules were but just for this file.

File: hostinfo/services.lua [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    function(cb)
+      scanDir(systemv, 'systemv', cb)
+    end,
+    function(cb)
+      scanDir(systemd[1], 'systemd', cb)
+    end,
+    function(cb)
+      scanDir(systemd[2], 'systemd', cb)
+    end
+    --[[,function(cb)
```
Comment (rphillips) [inline hostinfo/services.lua]: are we going to include this commented block?

**Comment by creationix** [comment]
+1

**Pull request merged**